PROVESFlightControllerReference/Components/PacketProcessor/docs/sdd.md

```
@@ -0,0 +1,202 @@
+
+- Security Header (6 bytes): SPI (2) + Sequence Number (4)
+- Space Packet Primary Header (6 bytes)
+- Space Packet Data Field (includes F Prime command)
+- Security Trailer (16-byte HMAC)
+
+Output packet layout (forwarded):
+
+- Space Packet Primary Header (6 bytes)
+- Space Packet Data Field
+
+### Additional resources
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)